python/ml4ir/base/model/relevance_model.py

```diff
@@ -431,7 +469,8 @@ def fit(
             where key is metric name and value is floating point metric value.
             This dictionary will be used for experiment tracking for each ml4ir run
         """
-
+        if self.feature_config.aux_label: #val_ranking_score_new_MRR
+            monitor_metric = 'val_' + self.output_name + '_' + monitor_metric
```

Comment:
Why is this needed?

Reply:
We need to specify the metric to be tracked in the validation set.